### PR TITLE
Updated translation catalog + French

### PR DESCRIPTION
**Pull request by claudep**

### CODE REVIEW

File: two_factor/locale/fr/LC_MESSAGES/django.po [hunk 1]
```diff
@@ -67,56 +69,97 @@ msgstr "fr"
...
+msgid "Call number %s"
+msgstr "Appeler le numéro %s"
+
+#: plugins/phonenumber/method.py:38
+msgid "We sent you a text message, please enter the token we sent."
+msgstr "Vous avez reçu un texto (SMS), veuillez entrer le jeton que nous avons envoyé."
+
+#: plugins/phonenumber/method.py:40 templates/two_factor/core/setup.html:37
+msgid "We are calling your phone right now, please enter the digits you hear."
+msgstr "Nous essayons de vous contacter au numéro fourni à l'instant. Veuillez entrer les chiffres que vous entendez."
```
Comment (Natureshadow) [inline two_factor/locale/fr/LC_MESSAGES/django.po:107]:
```suggestion
msgstr "Nous appelons votre téléphone à l'instant. Veuillez entrer les chiffres que vous entendez."
```

**Comment by claudep** [comment]
I just added the Finish translation as it was also updated.

**Comment by claudep** [comment]
@Natureshadow, thanks for your suggestion, OK to approve this?

**Comment by claudep** [comment]
This is probably the last PR that I would like to push before releasing 1.15.1 (apart from the release bump commit). Any review welcome!

**Comment by Natureshadow** [comment]
1.15.1 is a patch release to fix a critical issue.

Please do NOT postpone critical bugfixes for some translations.

While your request for help in the related issue is certainly legit, it has nothing to do with the bugfix - you can, and by all means should, tag a bugfix release even for one single change.

That said, let me repeat: Please do not postpone a critical bugfix in order to wait for a French translation!

**Comment by claudep** [comment]
Note that this is not MY package, I just decided to help because a) I'm using it b) It was missing love. So if you think there are things that could be better handled, please join and help!

**Review by peymanslh — APPROVED** [review]
Thanks,
LGTM!

**Comment by Natureshadow** [comment]
> So if you think there are things that could be better handled, please join and help!

OK, so is this an invitation to just tag a release myself next time :)?

**Comment by claudep** [comment]
> OK, so is this an invitation to just tag a release myself next time :)?

Why not? This is the spirit of Jazzband modules after all!

**Pull request merged**